eZ/Publish/Core/REST/Client/Values/User/User.php

```diff
@@ -35,6 +35,14 @@ public function getVersionInfo()
     }
 
     /**
+     * {@inheritdoc}
```

Comment:
AFAIK (my info may be a bit outdated), `{inheritDoc}` is inline tag but it's used just to copy description. Comment blocks are inherited by phpDoc anyway, so:
```php
/**
 * {@inheritDoc}
 */
```
Means: replace description with the same description (which make no sense). You can use `{@inheritDoc}` to append/prepend description. Like ie:
```php
/**
 * Hey, this is great!
 * {@inheritDoc}
 * But be careful...
 */
```
Other tags and spec details (like `@param` or `@return`) are inherited anyway, but you can replace them.
```php
/**
 * @return int
 */
```
On inherited method, this docblock will just replace methods `return` type.

There's no explicit way to tell phpDoc, do or do not inherit doc, yet. It's always inherited.
I did research about 2ys ago, so unless something changed, there's no point in inheriting docs explicitly (or telling to replace desc. with desc. like with `{@inheritDoc}`).